tests/chainer_tests/optimizer_hooks_tests/test_gradient_noise.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+
+import mock
+import numpy as np
+
+
+import chainer
+from chainer import cuda
+from chainer import optimizer_hooks
+from chainer import optimizers
+from chainer import testing
+from chainer.testing import attr
+
+
+class SimpleLink(chainer.Link):
+
+    def __init__(self, w, g):
+        super(SimpleLink, self).__init__(param=w.shape)
+        self.param.data = w
+        self.param.grad = g
+
+
+class TestOptimizerGradientNoise(unittest.TestCase):
```

Comment:
Now that it is not under `optimizer`, how about renaming it to `TestGradientNoise`?